ext/travisci/install_puppetserver.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+lein=lein2
+
+# Update a single dependency in a leiningen project.clj file.
+update_dependency_var() {
+    local file="$1"
+    local varname="$2"
+    local new_version="$3"
+
+    SED_ADDRESS="(def $varname"
+    SED_REGEX="\".*\""
+    SED_REPLACEMENT="\"$new_version\""
+    SED_COMMAND="s|$SED_REGEX|$SED_REPLACEMENT|"
+
+    set -e
+    set -x
+
+    sed -i -e "/$SED_ADDRESS/ $SED_COMMAND" $file
+}
+
+lein-pprint() {
+    export PUPPETSERVER_HEAP_SIZE=1G
```

Comment:
You mean
```sh
  PUPPETSERVER_HEAP_SIZE=1G "$lein" with-profile dev,ci pprint "$@" | sed -e 's/^"//' -e 's/"$//'
```
Didn't work?  And do we really need this just for pprint?  Hmm, Actually, there must be something else going on since this var is already exported in the envt via line 31 below.